metering/Dockerfile

```diff
@@ -0,0 +1,38 @@
+ARG IMAGE_VERSION=latest
```

Comment:
This file should be a symlink with Dockerfile -> Containerfile.

Reply:
changed to a symlink. Thanks for the hint.